lib/access-code-manager.js

```diff
@@ -22,6 +22,37 @@ module.exports = (config, promise, random, storageServiceFactory) => {
     var codeLength, secretLength, storageService;
 
     /**
+     * Verifies that an account ID is not empty.
+     *
+     * @param {string} accountId
+     * @return {Promise.<*>}
+     */
+    function checkAccountIdAsync(accountId) {
```

Comment:
This function is rewritten in account-manager, challenge-manager, and session-manager. Could it be moved into its own module to DRY things up?